tools/start_server.py

```diff
@@ -191,26 +236,37 @@ def getch():
 
         return ch
 
-def startServer(server):
-	print("HMI signals listener was started\r")
+def start_signals_listener(server):
 	server.set_fn_new_client(new_client)
 	server.set_fn_client_left(client_left)
 	server.set_fn_message_received(message_received)
 	server.run_forever()
 
-def keyBoardEvent():
-	global server
-	char = ' '
-	while char != 'q':
-		char = getch()
+def start_file_server(file_server):
+	file_server.serve_forever()
+
+def signal_handler(sig, frame):
+    print('\rStopping server...')
```

Comment:
```suggestion
    print("\rStopping server...")
```
only 2 places use single quotes

Reply:
@iCollin fixed in https://github.com/smartdevicelink/sdl_hmi/pull/274/commits/b506afc2d807d2ffa04860513272c8f6129f7f03